DEVELOPMENT.md

```diff
@@ -194,6 +217,9 @@ data:
   clusteringress.class: "istio.ingress.networking.knative.dev"
 ```
 
+You should keep the default value "*" for "istio.sidecar.includeOutboundIPRanges", when you use Minikube or
```

Comment:
you might need to escape the `*` otherwise the rest of the line./file is in italics, at least when viewed in github/md

Reply:
yep, interesting.